configs/foreman_118.yaml

```diff
@@ -0,0 +1,147 @@
+:project: foreman
+:prior_releases:
+  :1.17.0:
+    :redmine_release_id: 296
+  :1.17.1:
+    :redmine_release_id: 297
+:releases:
+  :1.18.0:
+    :redmine_release_id: 330
+:repos:
+  :foreman:
+    :main: true
+    :branch: 1.18-stable
+    :version_branch: true
+    :repo: https://github.com/theforeman/foreman.git
+  :smart-proxy:
+    :main: true
+    :branch: 1.18-stable
+    :version_branch: true
+    :repo: https://github.com/theforeman/smart-proxy.git
+  :foreman-selinux:
+    :main: true
+    :branch: 1.18-stable
+    :version_branch: true
+    :repo: https://github.com/theforeman/foreman-selinux.git
+  :foreman-installer:
+    :main: true
+    :branch: 1.18-stable
+    :version_branch: true
+    :repo: https://github.com/theforeman/foreman-installer.git
+  :foreman-packaging-deb:
+    :main: true
+    :branch: deb/1.18
+    :version_branch: true
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :foreman-packaging-rpm:
+    :main: true
+    :branch: rpm/1.18
+    :version_branch: true
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :community-templates:
+    :main: true
+    :branch: 1.18-stable
+    :version_branch: true
+    :repo: https://github.com/theforeman/community-templates.git
+
+:gpg_key: '57CEF41F'
+:mash_scripts:
+   - foreman-1.18-rhel7-dist
+   - foreman-plugins-1.18-rhel7-dist
+:tags:
+  - name: foreman-1.18-rhel7
+    based_off: foreman-nightly-rhel7
+    helper_tags:
+      foreman-1.18-rhel7-override: null
+    build_target: foreman-1.18-rhel7-build
+    build_package_group_source_tag: foreman-nightly-rhel7-build
+    arches:
+      - x86_64
+    inherits:
+      foreman-1.18-rhel7-build:
+        foreman-1.18-rhel7-override: 0
+        foreman-1.18-nonscl-rhel7: 10
+      foreman-1.18-rhel7-override:
+        foreman-1.18-rhel7: 0
+    external_repos:
+      - epel-7
+      - centos-sclo-rh-rhel-7
+      - centos-sclo-sclo-rhel-7
+      - centos-7-server
+      - rhel-7.0-server
+      - tfm-ror51-rhel-7
+  - name: foreman-1.18-nonscl-rhel7
+    based_off: foreman-nightly-nonscl-rhel7
+    helper_tags:
+      foreman-1.18-rhel7-override: null
```

Comment:
I got:
```
kkoji add-tag-inheritance --priority=0 foreman-1.18-nonscl-rhel7-build foreman-1.18-nonscl-rhel7-override
ERROR: Usage: kkoji add-tag-inheritance [options] tag parent-tag
(Specify the --help global option for a list of other help options)

kkoji: error: Invalid tag: foreman-1.18-nonscl-rhel7-override
```

so maybe this should be `foreman-1.18-nonscl-rhel7-override`?

Reply:
Ahh yup it should